beacon_node/src/cli.rs

```diff
@@ -261,6 +261,12 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
             .min_values(0)
             .hidden(true)
         )
+        .arg(
+            Arg::with_name("disable-backfill-rate-limiting")
+                .long("disable-backfill-rate-limiting")
+                .help("Disable the backfill sync rate-limiting. This allow users to just sync the entire chain as fast as they can.")
```

Comment:
```suggestion
                .help("Disable the backfill sync rate-limiting. This allow users to just sync the entire chain as fast \
                    as possible, however it can result in resource contention which degrades staking performance. Stakers \
                    should generally choose to avoid this flag since backfill sync is not required for staking.")
```

Just a suggestion to give users a bit more info.

Reply:
Thanks!